tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -0,0 +1,396 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(expected_final_state)
</pre><p>If you don't print it will be displayed prettier</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>